alerts_backend/python/app.py

```diff
@@ -23,40 +24,66 @@
 engine = create_engine(
     "sqlite+pysqlite:////var/db/aeroapi_alerts/aeroapi_alerts.db", echo=False, future=True
 )
+table_name = "aeroapi_alerts_table"
 
 
-def insert_into_db(data_to_insert: Dict[str, Union[str, int]]) -> int:
+@app.before_first_request
+def create_table():
     """
-    Insert object into the database based off of the engine.
-    Assumes data_to_insert has values for all the keys:
-    fa_alert_id, ident, origin, destination, aircraft_type, start_date, end_date.
+    Check if table exists, and if it doesn't create it.
     Returns 0 on success, -1 otherwise
     """
-    table_name = "aeroapi_alerts_table"
     try:
         metadata_obj = MetaData()
         # create the table if it doesn't exist
-        table_to_insert = Table(
+        table_to_create = Table(
             table_name,
             metadata_obj,
             Column("fa_alert_id", Integer, primary_key=True),
-            Column("ident", String(30)),
+            Column("ident", String(30), ),
             Column("origin", String(30)),
             Column("destination", String(30)),
             Column("aircraft_type", String(30)),
             Column("start_date", String(30)),
             Column("end_date", String(30)),
+            Column("max_weekly", Integer),
+            Column("eta", Integer),
+            Column("arrival", Boolean),
+            Column("cancelled", Boolean),
+            Column("departure", Boolean),
+            Column("diverted", Boolean),
+            Column("filed", Boolean),
+
         )
-        table_to_insert.create(engine, checkfirst=True)
+        table_to_create.create(engine, checkfirst=True)
+        app.logger.info("Table successfully created / updated")
+    except exc.SQLAlchemyError as e:
+        app.logger.error(f"SQL error occurred during creation of table (CRITICAL - INSERT WILL FAIL): {e}")
```

Comment:
This seems problematic enough that it's worth raising an exception instead of relying on the caller to check an error code